lib/plugins/tag/asset_link.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 const url = require('url');
+const util = require('hexo-util');
```

Comment:
suggest to refactor L4-5 to `const { escapeHTML } = require('hexo-util');`. Apply to "lib/plugins/tag/post_link.js " too.

Reply:
@curbengh thank you for review. I have fixed them.